src/components/FullscreenModal/FullscreenModal.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useRef } from 'react';
```

Comment:
is there for example a README.md next to these, to explain what is a full-screen modal and why/when should I use it? (part of storybook?)

Reply:
> is there for example a README.md next to these
 There is the Storybook file with all supported cases and you can see source code directly in Storybook Docs
 
 > why/when should I use it?
 That is more for UX team, they are aware when to use this pattern
 
But yes we should improve documentation of the components, but as that is work for weeks and it should be probably done in cooperation with UX, if this doesn't get some priority (and TBH it probably will not get any) I'd rather stay with the documentation provided automatically by SB.
I tried to do my best with documenting props and possible values in SB as well as adding "playground" stories (you can try combination of props directly in SB) to components where it makes sense.